studybuddies/src/app/services/auth.service.ts

```diff
@@ -1,24 +1,38 @@
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { environment } from '../../environments/environment';
+import { Observable } from 'rxjs';
 
 @Injectable({
   providedIn: 'root'
 })
 export class AuthService {
 
-  private URL = 'http://localhost:3000/api'
+  private httpHeaders: HttpHeaders = new HttpHeaders({
+    'Content-Type': 'application/json'
+  })
+  private urlLogin: string = '';
 
-  constructor(private http: HttpClient) { }
+  constructor(private http: HttpClient) {}
 
-  signUp(user: any){
-    return this.http.post<any>(this.URL + '/user/register', user);
+  getUrl(){
+    return this.urlLogin
   }
 
-  signIn(user: any){
-    return this.http.post<any>(this.URL + '/user/login', user);
-  }
+  /* signUp(user: any){
+    return this.http.post<any>(this.URL + '/user/register', user);
+  } */
 
-  loggedIn() {
-    return !!localStorage.getItem('token');
+  login(username: string, password: string): Observable<any>{
+    this.urlLogin = `${environment.urlBackend}${environment.rutaLogin}`
+    let data = {
+      username: username,
+      password: password
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
      password
```